app/src/test/java/org/mozilla/focus/utils/UrlUtilsTest.kt

```diff
@@ -0,0 +1,173 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils
+
+import android.annotation.SuppressLint
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.focus.browser.LocalizedContent
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class UrlUtilsTest {
```

Comment:
You converted the test file to Kotlin but it seems like there are no tests that cover the new methods? :)